### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by maprfsousa** [rustdesk/rustdesk#546] [General]
Rustdesk doesn't work on w7, I can't install. Sakta out. Thanks.

**Comment by rustdesk** [comment]
Thanks, some one reported also. But I can not reproduce on my w7. Our installation is a simple script calling, Wondering if you can help figure out the problem.

https://github.com/rustdesk/rustdesk/blob/master/src/platform/windows.rs#L972
